index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+  <header class="header-container">
+    <div class="header-item header-item1"><time id="time">9:41</time></div>
+    <div class="header-item header-item2"><img id="signal" src="src/signal.png" alt="signal"></div>
```

Comment:
Kindly note that though this is included in the wireframe, its not necessary. The only parts of the wireframe that are needed are the logo, which is your name and the hamburger icon which you will use later to display the menu items. This are just guides to show you its in mobile. Kindly delete them.